### DISCUSSION TITLE
ERD

### DISCUSSION THREAD

**Discussion by thirdygayares** [Jobboardplatform-svfc/JobBoardPlatform_Documentation#8] [Ideas]
hindi dapat plural ang pangalan ng table

![image](https://github.com/user-attachments/assets/e58587b2-6378-43ba-bdc3-2b2e8643eaae)

**Comment by arjaycano** [comment]
Noted po Sir, btw edit ko na rin po ba sa docu namin yung changes mo po Sir
sa Lucidchart namin?
